src/examples/scala/imputeParallelCampAll.scala

```diff
@@ -0,0 +1,68 @@
+object imputeParallelCampAll extends loamstream.LoamFile {
+  //---------------------------GLOBAL SETTINGS----------------------------------------------------------------------------
+  val kgp_dir = path("/humgen/diabetes/users/ryank/data/1kg_phase3/1000GP_Phase3")
+  
+  val software_dir = path("/humgen/diabetes/users/ryank/software")
+  val shapeit = software_dir / "shapeit/bin/shapeit"
+  val impute2 = software_dir / "impute_v2.3.2_x86_64_static/impute2"
+  
+  val shapeit_data_dir =  path("/humgen/diabetes/users/ryank/data/camp/data_clean/split")
+  val output_dir = path("/humgen/diabetes/users/kyuksel/imputation/run_dir")
+  
+  val num_bases_per_shard = 1000000
+  
+  // Map: Chrom Number -> (Number of Shards, Offset for Start Position)
+  var chr_props: Map[Int, (Int, Int)] = Map.empty
+  chr_props += (1 -> (250, 10176))
+  chr_props += (2 -> (244, 10178))
+  chr_props += (3 -> (198, 60068))
+  chr_props += (4 -> (192, 10004))
+  chr_props += (5 -> (182, 10042))
+  chr_props += (6 -> (172, 63853))
+  chr_props += (7 -> (160, 14807))
+  chr_props += (8 -> (147, 11739))
+  chr_props += (9 -> (142, 10162))
+  chr_props += (10 -> (137, 60493))
+  chr_props += (11 -> (136, 61394))
+  chr_props += (12 -> (135, 60180))
+  chr_props += (13 -> (97, 19020046))
+  chr_props += (14 -> (89, 19000016))
+  chr_props += (15 -> (83, 20000040))
+  chr_props += (16 -> (91, 60085))
+  chr_props += (17 -> (82, 51))
+  chr_props += (18 -> (79, 10082))
+  chr_props += (19 -> (60, 60841))
+  chr_props += (20 -> (64, 60342))
+  chr_props += (21 -> (39, 9411238))
+  chr_props += (22 -> (36, 16050074))
+  
+  //---------------------------LOOP THROUGH CHROMOSOMES-------------------------------------------------------------------
```

Comment:
Examples are starting to get long with multiple stage sections. I assume function `def`s are completely allowed given that it's pure Scala code. Might be good to show use of them just as a reminder to users.

If there are any issues w/ using functions (e.g. `cmd`s behaving in unexpected ways), then that would be excellent to comment on/document as well.